storage/Filesystems/NtfsImpl.h

```diff
@@ -54,7 +54,7 @@ namespace storage
 	virtual bool supports_unmounted_grow() const override { return true; }
 
 	virtual bool supports_label() const override { return true; }
-	virtual unsigned int max_labelsize() const override { return 128; }
+	virtual unsigned int max_labelsize() const override { return 32; }
```

Comment:
It would have been nice to have the rationale with all those pointers somewhere close to this 32. Or at least a pointer to the bug report and add the information there.

So far, the useful information and the links to all the info sources are only in the pull request description, which is not the most trustworthy information in the long term (neither the easier to find while debugging).

Reply:
That Stack Exchange post contained a link to a Microsoft site, but that link has become invalid in the meantime.

I also would have preferred to have a more authoritative source, preferably from Microsoft themselves, but I couldn't find anything about that NTFS label length on their web pages; but there are literally dozens of third-party sites all claiming it's just 32.

So I chose to remain on the conservative side and rather use the shorter limit. It's not that this is a painful restriction; most other filesystem types have a much shorter max size (typically 12-16). The only exception is Btrfs.

The most prominent use of those labels from our perspective is in /etc/fstab anyway, and more than 32 seems pretty excessive to me already.